tools/p4dbg.py

```diff
@@ -676,6 +678,8 @@ def handle_config_change(self):
         print "or we will exit."
         v = prompt_yes_no("Do you want to request the new config?", True)
         if v:
+            if not self.standard_client:
```

Comment:
There is no sense in even asking the user, since you won't be able to load the new config. You should just print something and exit with no prompt.

Reply:
Good point. Updated. It now prints that it cannot connect via thrift to reload the config, and exits.